### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by caco3** [haverland/collectmeteranalog#8]
The readme does not mention any system requirements.

As far as I can see, it currently does not support Windows.
It would not be too hard to make it work on windows, I can help if desired.

Also I think for users not having Python installed, we could provide a standalone binary. I have done that before and its not too hard. See https://setuptools.pypa.io/en/latest/setuptools.html

**Comment by haverland** [comment]
I have no Windows. So I can not test it. Why should windows not supported?

I you could provide it, would be nice.

**Comment by caco3** [comment]
When I tried to run it on Windows, I had an error related to a Unix path (`./data`). How ever today it worked fine, but I run it differently.

**Comment by caco3** [comment]
I looked into building an executabel version (Windows or Linux).

With `pyinstaller` this is easily possible, eg. running following (on Linux):
```
cd collectmeteranalog
pyinstaller --onefile collectmeteranalog/__main__.py
```
the Linux executable then is placed in `collectmeteranalog/dist/__main__`

To run it, one must provide the model: `./collectmeteranalog/dist/__main__  --collect=192.168.1.10 --days=3 --model ./collectmeteranalog/models/ana-class100_0120_s1_q.tflite`

How ever the executable is over 600 MB in size, so I don't think it feasable :(

Beside of the size, it might be possible to build windows an Linux binaries for free using the Github actions.

**Comment by haverland** [comment]
I' am on it as GitHub-action too. 

600MB is a bit much. The GitHub action creates a 50MB zip, but didn't work yet.
./data moved to data/ and should work on windows now (not tested yet)

**Comment by haverland** [comment]
https://stackoverflow.com/questions/47692213/reducing-size-of-pyinstaller-exe

The size depends on all installed packages. Clean environment reduces the size.

**Comment by caco3** [comment]
It gets as large because it integrates all required python libraries:
![grafik](https://user-images.githubusercontent.com/1783586/186269069-053de456-a1c3-4726-b2d6-78f0976619ac.png)

How did you get it down to 50 MB?

I tried in a venv, but still 600 MB. according to above screenshot, tensorflow alone has 880 MB (uncompressed).

**Comment by haverland** [comment]
Included in 1.0.0. Uses tflite-runtime now instead of TensorFlow. But windows and macOS will not have a prediction.
